index.html

```diff
@@ -151,6 +151,8 @@ <h2 class="title">Multi-Post Stories <br> Gain+Glory</h2>
     <section id="about">
       <div class="frameleft"><img src="./Images/frameleft.svg" alt=""></div>
         <div class="frameright"> <img src="./Images/frameright.svg" alt=""></div>
+        <div class="frameleftdesk"><img src="./Images/frameleftdesktop.svg" alt=""></div>
+        <div class="framerightdesk"> <img src="./Images/framerightdesktop.svg" alt=""></div>
```

Comment:
- [ ] to match the Figma design kindly update your framework's card size (height& width) to be equal to each other , and please reduce the card title size 🙏 

your design 
![image](https://user-images.githubusercontent.com/89282221/214825581-fcb9fd91-db7b-427c-9f54-71a494e6fcc1.png)

figma 

![image](https://user-images.githubusercontent.com/89282221/214825639-f6e2b135-7bb0-467b-aa31-c7ff134783f3.png)